Assets/Scripts/Movements/AerialMovement.cs

```diff
@@ -0,0 +1,37 @@
+using UnityEngine;
+
+// Aerial movement.
+//
+// Interceptors and threats have different movements since threats are not affected by drag and
+// gravity and have specified attack behaviors.
+public abstract class AerialMovement : MovementBase {
+  public AerialMovement(IAgent agent) : base(agent) {}
+
+  // Calculate the dynamic pressure on the agent.
+  protected float GetDynamicPressure() {
+    var airDensity = Constants.CalculateAirDensityAtAltitude(Agent.Position.y);
+    var flowSpeed = Agent.Speed;
+    return 0.5f * airDensity * (flowSpeed * flowSpeed);
+  }
+
+  // Calculate the air drag acting on the agent.
+  protected float CalculateDrag() {
+    var staticConfig = Agent.StaticConfig;
+    var dragCoefficient = staticConfig.LiftDragConfig?.DragCoefficient ?? 0;
+    var crossSectionalArea = staticConfig.BodyConfig?.CrossSectionalArea ?? 0;
+    var mass = staticConfig.BodyConfig?.Mass ?? 1;
+    var dynamicPressure = GetDynamicPressure();
+    var dragForce = dragCoefficient * dynamicPressure * crossSectionalArea;
+    return dragForce / mass;
```

Comment:
Division by zero if mass is zero. Should we protect? Should we test? Should we somewhere clamp mass to be 1 epsilon > 0?